libminifi/src/utils/ProcessCpuUsageTracker.cpp

```diff
@@ -126,7 +127,7 @@ double ProcessCpuUsageTracker::getProcessCpuUsageBetweenLastTwoQueries() const {
   uint64_t sys_cpu_times_diff = sys_cpu_times_ - previous_sys_cpu_times_;
   uint64_t user_cpu_times_diff = user_cpu_times_ - previous_user_cpu_times_;
   double percent = static_cast<double>(sys_cpu_times_diff + user_cpu_times_diff) / static_cast<double>(cpu_times_diff);
-  percent = percent / std::thread::hardware_concurrency();
+  percent = percent / (std::max)(static_cast<uint32_t>(1), std::thread::hardware_concurrency());
```

Comment:
`uint32_t{1}` is shorter and avoids the cast.

Reply:
Updated in a199f6907d09938d30af9fe21b65ffc0e5116cb2